examples/local/env.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
A bit of a nitpick but not sure if you are using here any bash-specific feature, could you just use `sh` here?